Shared/Tracking/Trackers/myanimelist/MyAnimeListTracker.swift

```diff
@@ -31,6 +31,8 @@ class MyAnimeListTracker: OAuthTracker {
 
     var oauthClient: OAuthClient { api.oauth }
 
+    var restrictSources: [String]? { nil }
```

Comment:
this should be defined as a default for trackers, under where `scoreOptions` is set in Tracker.swift, so you don't have to define it in all trackers, just trackers that are actually restricted to certain sources